Automattic/wp-calypso
# Add warning notice in checkout when plan includes backup already

#### Changes proposed in this Pull Request

Customers who purchased a Backup product (monthly or real time) can have a duplicated backup feature by purchasing a Jetpack plan that includes it as well. This PR adds a warning for such cases in the checkout screen.

#### Testing instructions

1. No Backup
    * Select a Jetpack site that has no Backup
    * Go to _Plan > _Plans_
    * Select any of the 3 plans
    * Once in the checkout page, verify that you **don't** see any notice and that there's no error in the console
    * Repeat for the other plans
2. Backup Daily
    * Select a Jetpack site that has Backup Daily
    * Go to _Plan > _Plans_
    * Select any of the 3 plans
    * Once in the checkout page, verify that you **do** see a notice and that there's no error in the console
    * Both links in the notice should redirect you to the Backup Daily purchase page
    * Repeat for the other plans
3. Backup Real-Time
    * Select a Jetpack site that has Backup Real-Time
    * Go to _Plan > _Plans_
    * Select any of the 3 plans
    * Once in the checkout page, verify that you **do** see a notice **only for the Business plan**
    * Both links in the notice should redirect you to the Backup Real-Time purchase page
    * Repeat for the other plans

#### Screenshots

**Mockups**
<img width="809" alt="Screen Shot 2020-07-16 at 3 32 29 PM" src="https://user-images.githubusercontent.com/1620183/87714424-bfbc9980-c779-11ea-86da-7258a4fa1905.png">

**Implementation**
<img width="746" alt="Screen Shot 2020-07-16 at 3 33 03 PM" src="https://user-images.githubusercontent.com/1620183/87714419-bcc1a900-c779-11ea-80de-40ceb1136848.png">
